src/manager/defaultWallpaperManager.ts

```diff
@@ -54,8 +54,7 @@ class DefaultWallpaperManager extends WallpaperManager {
             // merged wallpapers need mode "spanned"
             backgroundSettings.setString('picture-options', 'spanned');
         else
-            // single wallpapers need mode "zoom"
-            backgroundSettings.setString('picture-options', 'zoom');
+            backgroundSettings.setString('picture-options', getZoomName(new Settings().getInt('zoom-mode')));
```

Comment:
The downside is that we couldn't reset to the previously selected state here without duplicating the option in our schema.